### PR TITLE
openapi: return reana_server_version

### PR DESCRIPTION
**Pull request by ParthS007**
closes https://github.com/reanahub/reana-client/issues/441

### CODE REVIEW

File: reana_commons/openapi_specifications/reana_server.json [hunk 1]
```diff
@@ -2466,6 +2466,7 @@
               "application/json": {
                 "email": "user@reana.info",
                 "full_name": "John Doe",
+                "reana_server_version": "x.yy.zz",
```
Comment (tiborsimko) [inline reana_commons/openapi_specifications/reana_server.json:2469]: It would be good to use a real number for the best illustration, e.g. "0.7.0".

Comment (ParthS007) [reply reana_commons/openapi_specifications/reana_server.json]: Done 👍

File: reana_commons/openapi_specifications/reana_server.json [hunk 2]
```diff
@@ -2479,6 +2480,9 @@
                 "email": {
                   "type": "string"
                 },
+                "reana_server_version": {
+                  "type": "string"
+                },
```
Comment (tiborsimko) [inline reana_commons/openapi_specifications/reana_server.json:2485]: Please rebase against latest `maint-0.7`, something like:

```console
$ git checkout maint-0.7
$ git fetch upstream
$ git merge --ff-only upstream/maint-0.7
$ git checkout 144
$ git rebase maint-0.7 -i
```

Comment (tiborsimko) [reply reana_commons/openapi_specifications/reana_server.json:2485]: BTW note that you can also catch these locally before pushing a PR set for review in the following manner:

```console
$ reana-dev git-fetch -c .
$ reana-dev git-status -c . --base maint-0.7
reana-commons @ pr-231 (STEMS FROM 7 BEHIND upstream/maint-0.7) @ 6cb9afa openapi: return reana_server_version
``` 

The warnings in red such as the given branch stems from behind upstream etc may be printed if something is not in order.

Comment (ParthS007) [reply reana_commons/openapi_specifications/reana_server.json:2485]: Done 👍

**Review by diegodelemos — APPROVED** [review]

File: CHANGES.rst [hunk 3]
```diff
@@ -1,11 +1,13 @@
 Changes
 =======
 
-Version 0.7.1 (UNRELEASED)
+Version 0.7.1 (2020-11-09)
 --------------------------
 
 - Adds support for restarting yadage workflows (through ``accept_metadir`` operational option).
 - Allows ``htcondor_max_runtime`` and ``htcondor_accounting_group`` to be specified for HTC jobs.
```
Comment (tiborsimko) [inline CHANGES.rst:8]: Leading "Allows" is not following our "Adds/Fixes/Changes" triad. Should rather be "Adds support for specifying ...".

Also, HTC would beter be spelled HTCondor in full.

**Pull request merged**